packages/editor/src/components/post-template/create-new-template-modal.js

```diff
@@ -92,7 +96,7 @@ export default function CreateNewTemplateModal( { onClose } ) {
 			] );
 
 		const newTemplate = await createTemplate( {
-			slug: cleanForSlug( title || DEFAULT_TITLE ),
+			slug: 'wp-custom-template-' + kebabCase( title || DEFAULT_TITLE ),
```

Comment:
```suggestion
			slug: kebabCase( title || DEFAULT_TITLE ) || 'wp-custom-template',
```

This would avoid prefixing the custom template when the title is with Latic chars. What do you think?

Reply:
It's a good idea, because the template part creation uses a similar logic:

https://github.com/WordPress/gutenberg/blob/f12ba660ed2fb14766f81716fee7fd0d1ce10103/packages/fields/src/components/create-template-part-modal/utils.js#L66-L68

In f12ba660ed2fb14766f81716fee7fd0d1ce10103, I've also updated the logic in the site editor for consistency, what do you think?

Reply:
Thanks, @t-hamano!